en/core-libraries/text.rst

```diff
@@ -38,6 +38,52 @@ of a ``View``, use the ``Text`` class::
         }
     }
 
+
+Transliterate Strings
+=====================
+
+.. php:staticmethod:: transliterate($string, $transliteratorId = null)
+
+Transliterate by default converts all characters in provided string into
+equivalent ASCII characters. The transliterate method expects UTF-8 encoding.
+You can use the ``$transliteratorId`` argument to control the transliteration.
+Using ``Text::setTransliteratorId()`` you can change the default transliteration
+id. Examples::
+
+    // apple puree
+    Text::transliterate('apple purée');
+
+    // Ubermensch люблю
```

Comment:
By default it would but in this example transliteration id used `'Latin-ASCII;'` tells it convert latin chars to ASCII. ICU transliteration identifiers are basically of form `<source script>:<target script>` and you can specify multiple conversion pairs separated by `;`. More info [here](http://userguide.icu-project.org/transforms/general).